docs/makeqbk.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Making a note that @ximinez Checked both cygwin and the git shell. This shebang works.